colony_picker/utils.py

```diff
@@ -146,52 +146,6 @@ def view_projection(points_3d: np.ndarray,
     return plotter
 
 
-def generate_fiducials(
-        tag_dictionary: cv2.aruco_Dictionary,
-        ids: np.ndarray,
-        size: float,
-        base_file_name: str,
-        include_white_border: bool = False,
-        dpi: float = 300):
-    """Generate a set of Aruco tag fiducial images and save them as files.
```

Comment:
The docstring is great! Also, the function + variable names very clearly describe their purpose which is great!